packages/components/src/components/div100vh-container/div100vh-container.tsx

```diff
@@ -38,7 +38,7 @@ const Div100vhContainer = ({
         height: max_autoheight_offset ? '' : height_rule,
         maxHeight: max_autoheight_offset ? `calc(100rvh - ${max_autoheight_offset})` : '',
     };
-    if (is_bypassed) return children;
+    if (is_bypassed) return children as JSX.Element;
```

Comment:
```suggestion
    if (is_bypassed) return <>children</>;
```
can you check this please? also you might need to check the usage of the component as well to make sure it's working as expected ( since it getting wrapped inside fragment )